tests/celery_tasks_test.py

```diff
@@ -16,8 +16,12 @@
 
 class CeleryTasksTestCase(unittest.TestCase):
     def setUp(self):
+        self.image_url = 'https://www.screenly.io/upload/ose-logo.png'
```

Comment:
Let's remove this as I don't think it exists anymore. Perhaps use something from the GitHub repo instead.

Reply:
Alright, thanks! Will have to use the following instead: https://github.com/Screenly/screenly-ose/raw/master/static/img/ose-logo.png